guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -27,6 +28,18 @@ endif::[]
 ----
 # {package-install-project} python3-pulp_manifest
 ----
+ifdef::satellite[]
++
+Note that this command stops the {Project} service and re-runs {foreman-installer}.
+Alternatively, to prevent downtime caused by stopping the service, you can use the following:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages unlock
+# {package-install} python3-pulp_manifest
```

Comment:
again, I am unsure about the package name. Could you please provide a link to yum.theforeman.org for this package? Or can someone else confirm?

Reply:
@lpramuk, could you please help here? 
Thanks!

Reply:
It is indeed a problem that it isn't published in upstream. That was intended to be fixed. It's a very valid point because we do have packages with a `python38` and `python39` prefix so it's quite crucial to get it right.

Reply:
I'm not sure what fix should be here in that case, Can someone guide me here if anything is pending from my side? Thanks!

Reply:
I've opened an issue against pulpcore-packaging: https://github.com/theforeman/pulpcore-packaging/issues/598

My understanding is that once the Pipeline ran through, the built package will be available at http://yum.theforeman.org/pulpcore/ where you can get the package name from.

Reply:
@dubewarsagar Pls, correct the package names in both modules. The correct package names for 6.11 are:

- on EL7: `tfm-pulpcore-python3-pulp_manifest`
- on EL8: `python38-pulp_manifest`

(I checked in Ohsnap and I'm assuming it corresponds with pulpcore 3.16)

Reply:
Updated package names in both modules.

Reply:
@Lennonka updated package names in both the modules. 
@maximiliankolb pls check now and in case it is good to merge, kindly do the needful. 

Thank you both :-)